src/compiler/checker.ts

```diff
@@ -12744,6 +12940,13 @@ namespace ts {
             let isDecorator: boolean;
             let spreadArgIndex = -1;
 
+            if (isJsxOpeningLikeElement(node)) {
+                // For JSX opening-like element, we will ignore regular arity check (which is what is done here).

```

Comment:
can you make this comment shorter? Basically just give a link to the function that does the work instead: "The arity check will be done in `checkApplicableSignatureForJsxOpeningLikeElement` instead".